dev-docs/bidders/hypelab.md

```diff
@@ -0,0 +1,19 @@
+---
+layout: bidder
+title: HypeLab
+description: Prebid HypeLab Bidder Adapter
+pbjs: true
+pbs: false
+biddercode: hypelab
+sidebarType: 1
+media_types: banner
+userIds: all
```

Comment:
There are a few properties missing that help readers to vet your bidder

```
safeframes_ok: true/false
floors_supported: true/false
fpd_supported: true/false
ortb_blocking_supported: true/partial/false
```

Reply:
Done!